src/components/idenfy/IdenfyPingUserStatus.tsx

```diff
@@ -14,8 +14,12 @@ import {
 
 const IdenfyModal: FunctionComponent = () => {
   const dispatch = useDispatch();
+  const { isLoggedIn } = useSelector(selectSession);
 
-  const { data: { verificationStatus } = emptyProfile } = useGetProfileQuery();
+  const { data: { verificationStatus } = emptyProfile } = useGetProfileQuery(
+    undefined,
+    { skip: !isLoggedIn }
```

Comment:
This was the culprit of us getting that error toast on login.
Reason, since this is an app level component it triggers a fetch profile data even if user is not logged in, we need to skip it if that is the case.